pkg/controller/bastion/options.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bastion
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/gardener/gardener-extension-provider-gcp/pkg/gcp"
+	"github.com/gardener/gardener/extensions/pkg/controller"
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	"github.com/gardener/gardener/pkg/extensions"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Options contains provider-related information required for setting up
+// a bastion instance. This struct combines precomputed values like the
+// bastion instance name with the IDs of pre-existing cloud provider
+// resources, like the Firewall name, subnet name etc.
+type Options struct {
+	Shoot               *gardencorev1beta1.Shoot
+	BastionInstanceName string
+	FirewallName        string
+	PublicIP            []string
```

Comment:
plural - rename to `PublicIPs`

Reply:
committed in c1472d2ede0f5d6334526fa6fb08c4e719c43b4b

Reply:
could you double check? You renamed it to `PublicIPs` which was my first suggestion, but then I changed the suggestion to `CIDRs`

Reply:
understood,  I have thought it was two different variables. will change it

Reply:
committed in fbad5cfe22865b73a53ff0684b28bd8563af8e5d

Reply:
committed in fbad5cfe22865b73a53ff0684b28bd8563af8e5d